migrations/alembic.ini

```diff
@@ -0,0 +1,50 @@
+# A generic, single database configuration.
```

Comment:
Might be worth adding some more explanation here as not sure why some of these attributes exist.

Reply:
This file was autogenerated. They are the logging configuration for when migrations are run.

Reply:
Loggers represent the different components that could be logging while running the migration: sqlalchemy, alembic, flask_migrate or root (main app).
Each logger defines level of log (WARN, etc) and handlers (what to do with the message). The messages will go to the console as they are defined now.